ansible/roles/software/ckan/catalog/ckan-app/templates/catalog-next/etc_ckan_production.ini.j2

```diff
@@ -188,7 +188,7 @@ ckan.hide_activity_from_users = %(ckan.site_id)s
 #smtp.user = your_username@gmail.com
 #smtp.password = your_password
 smtp.mail_from = {{ catalog_ckan_email_from }}
-
+smtp.test_server = localhost:6675
```

Comment:
Can you add a new Ansible variable `ckan_email_enabed: true` and then we can wrap this in an `if`
```jinja2
{% if ckan_email_enabled %}
smpt.test_server = true
{% endif %}
```

Reply:
Fixed